cms/djangoapps/contentstore/views/tests/test_group_configurations.py

```diff
@@ -829,6 +829,39 @@ def test_can_handle_without_parent(self):
         actual = GroupConfiguration.get_content_experiment_usage_info(self.store, self.course)
         self.assertEqual(actual, {0: []})
 
+    def test_can_handle_multiple_partitions(self):
+        # Create the user partitions
+        self.course.user_partitions = [
+            UserPartition(
```

Comment:
Can you please update the test so that `course.user_partitions` contains another user partition with scheme other than `cohort`,  e.g., `random` and check that `get_content_groups_usage_info` method returns only `cohort` user partition groups.